agent/workerlet/pathWalker/pathWalker.go

```diff
@@ -300,7 +300,8 @@ func (tm *taskMain) WalkPathTask(req workerlet.WalkPathRequest) {
 func (tm *taskMain) WalkPackageTask(req workerlet.WalkGetPackageRequest) {
 	var data share.ScanData
 	scanUtil := scan.NewScanUtil(tm.sys)
-	data.Buffer, data.Error = scanUtil.GetRunningPackages(req.Id, req.ObjType, req.Pid, req.Kernel, req.ImageRepo, req.PidHost)
+	data.Buffer, data.Error = scanUtil.GetRunningPackages(req.Id, req.ObjType, req.Pid,
```

Comment:
Could you rename the `imageRepo` in `scanUtil.GetRunningPackages()`'s parameters as well?